src/platform/packages/shared/kbn-ftr-common-functional-services/services/security/test_user.ts

```diff
@@ -23,15 +21,13 @@ export class TestUser extends FtrService {
   private readonly config = this.ctx.getService('config');
   private readonly log = this.ctx.getService('log');
 
-  private readonly browser: Browser | void =
+  private readonly browser =
     // browser service is not normally available in common.
-    this.ctx.hasService('browser') ? (this.ctx.getService('browser' as any) as Browser) : undefined;
+    this.ctx.hasService('browser') ? this.ctx.getService('browser' as any) : undefined;
```

Comment:
The construction here was not ideal: the service tries to determine if it's loaded for a UI suite and in that case adds steps in the UI after the test user role has been updated. Importing the types from the UI services would introduce a cyclic dependency. I've decided to remove the typing here as it's only a few lines of code making use of it. The alternative would probably be to move the browser and test subject services to yet another package so we can import from this common package as well as from the UI package, but to me that seemed too much effort for the gained benefit.